spec/smoke/api/v1/providers_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "spec_helper_smoke"
+
+describe "V1 Public API Smoke Tests", :aggregate_failures, smoke: true do
+  let(:course_year) { Time.current.year }
```

Comment:
```suggestion
  let(:recruitment_year) { Settings.current_recruitment_cycle_year }
```

The recruitment cycle year value changes around October time.